lab_5_scrapper/scrapper.py

```diff
@@ -195,6 +352,11 @@ def parse(self) -> Union[Article, bool, list]:
         Returns:
             Union[Article, bool, list]: Article instance
         """
+        response = make_request(self.full_url, self.config)
```

Comment:
you are so sure that request is always successful..

Reply:
hoped for the best... fixed